docs/content/guides/accessories-and-menus/empty-data-state/empty-data-state.md

```diff
@@ -0,0 +1,166 @@
+---
+id: empty-data-state-guide
```

Comment:
That value should match the other values on other pages. See https://github.com/handsontable/handsontable/blob/develop/docs/README-EDITING.md#frontmatter